recipes/wg21-linear_algebra/all/conanfile.py

```diff
@@ -35,39 +35,42 @@ def _minimum_compilers_version(self):
             "apple-clang": "11"
         }
 
+    @property
+    def _full_compiler_version(self):
```

Comment:
This is only used to print... would it not be easier to give user the same setting they are passing?

it seems more confusing to format it different-- then just a one to on matching?

Reply:
Fair point, this should now be simplified.  But let me know if it needs further clarification.